daily-patch-information/src/main/java/org/wso2/patchinformation/comparators/JIRADateComparator.java

```diff
@@ -0,0 +1,15 @@
+package org.wso2.patchinformation.comparators;
+
+import org.wso2.patchinformation.pmt.InactivePatch;
+
+import java.io.Serializable;
+import java.util.Comparator;
+/**
+ * Implements the Comparator class to order objects of the OpenPatch class by it's "jiraCreateDate" attribute.
+ */
+public class JIRADateComparator implements Comparator<InactivePatch>, Serializable {
```

Comment:
Why are we having so much of comparators, and moreover it's compared with dates where the date comparator is defined by default in Java, so why yet another specialized comparator?